F21/vineethreddyallam/Code Review.txt

```diff
@@ -0,0 +1,71 @@
+
+package Review;
+
+import java.util.*;
+
+import java.util.Locale;
+
+public class Main
+{
+	public static void main(String[] args) {
+	    //English language
+	            String strUpper, strLower;
+        Scanner scan = new Scanner(System.in);
+        

```

Comment:
Can use a choice or switch case to do perform each activity  in a simple way

Reply:
Sure buddy, I'll make it